js/src/lib/components/InferenceWidget/widgets/AudioClassificationWidgetV2/AudioClassificationWidgetV2.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import type { WidgetProps, WidgetOutput, WidgetInput } from "../../shared/types";
+
+	import WidgetWrapper from "../../shared/WidgetWrapper/WidgetWrapperV2.svelte";
+	import WidgetInputAudio from "../../widgetsInput/WidgetInputAudio/WidgetInputAudio.svelte";
+	import WidgetOutputClassification from "../../wigdetsOutput/WidgetOutputClassification/WidgetOutputClassification.svelte";
+
+	export let apiToken: WidgetProps["apiToken"];
+	export let apiUrl: WidgetProps["apiUrl"];
+	export let callApiOnMount: WidgetProps["callApiOnMount"];
+	export let model: WidgetProps["model"];
+	export let noTitle: WidgetProps["noTitle"];
+	export let includeCredentials: WidgetProps["includeCredentials"];
+
+	export let widgetInput: WidgetInput;
+	export let widgetOutput: WidgetOutput;
+	export let isLoading = false;
+	export let getOutput: ({withModelLoading = false,isOnLoadCall = false,}) => Promise<void>;
+</script>
+
+<WidgetWrapper
+	{apiToken}
+	{callApiOnMount}
+	{apiUrl}
+	{includeCredentials}
+	{isLoading}
+	{model}
+	{noTitle}
+	{widgetInput}
+	{widgetOutput}
+	bind:getOutput
+>
+	<svelte:fragment slot="input" let:isDisabled>
+		<WidgetInputAudio bind:widgetInput {isLoading} {isDisabled} on:click={() => getOutput()} />
+	</svelte:fragment>
+	<svelte:fragment slot="output">
+		<WidgetOutputClassification bind:widgetOutput />
+	</svelte:fragment>
+</WidgetWrapper>
```

Comment:
This is nice :+1:

I think it makes sense to fatcorize into inputs and outputs groups